pkg/kubectl/explain.go

```diff
@@ -38,8 +37,7 @@ func SplitAndParseResourceRequest(inResource string, mapper meta.RESTMapper) (st
 }
 
 // PrintModelDescription prints the description of a specific model or dot path
```

Comment:
Can you add a comment to the doc string explaining what the recursive argument does?